ruff.toml

```diff
@@ -37,7 +37,14 @@ target-version = "py311"
 [lint]
 # Enable Pyflakes (`F`) and a subset of the pycodestyle (`E`)  codes by default.
 select = ["E4", "E7", "E9", "F"]
-ignore = []
+ignore = [
+    # imported but unused
+    # __init__ often imports to make available
+    # but does not use
+    "F401",
+    # bare except
+    "E722",
```

Comment:
These 2 are where the most lint warnings are, so keeping these intact for now, and hoping to address the "unused imports" in a future PR.